website/addons/s3/templates/s3_node_settings.mako

```diff
@@ -77,3 +77,17 @@
         <p data-bind="html: message, attr.class: messageClass"></p>
     </div>
 </div>
+
+ <script>
+    // Make s3 settings variables globally accessible for relevant JS
+    <%
+        import json
+        from website.addons.s3.settings import DEFAULT_BUCKET_LOCATION
+    %>
+    window.contextVars = $.extend(true, {}, window.contextVars, {
+        s3Settings: {
+            defaultBucketLocationValue: ${json.dumps(DEFAULT_BUCKET_LOCATION['value'])},
```

Comment:
@abought I was able to reproduce the error locally, even after getting this branch up to date with develop.

```
Traceback (most recent call last):
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/sloria/projects/cos-projects/osf/framework/routing/__init__.py", line 112, in wrapped
    data = fn(*args, **kwargs)
  File "/Users/sloria/projects/cos-projects/osf/website/views.py", line 360, in resolve_guid
    return proxy_url(url)
  File "/Users/sloria/projects/cos-projects/osf/framework/routing/__init__.py", line 266, in proxy_url
    response = app.view_functions[match[0]](**match[1])
  File "/Users/sloria/projects/cos-projects/osf/framework/routing/__init__.py", line 125, in wrapped
    return renderer(data, **renderer_kwargs or {})
  File "/Users/sloria/projects/cos-projects/osf/framework/routing/__init__.py", line 330, in __call__
    rendered = self.render(data, redirect_url, *args, **kwargs)
  File "/Users/sloria/projects/cos-projects/osf/framework/routing/__init__.py", line 613, in render
    return self._render(data, template_name)
  File "/Users/sloria/projects/cos-projects/osf/framework/routing/__init__.py", line 546, in _render
    rendered = renderer(self.template_dir, template_name, data, trust=self.trust)
  File "/Users/sloria/projects/cos-projects/osf/framework/routing/__init__.py", line 235, in render_mako_string
    return tpl.render(**data)
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/mako/template.py", line 443, in render
    return runtime._render(self, self.callable_, args, data)
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/mako/runtime.py", line 803, in _render
    **_kwargs_for_callable(callable_, data))
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/mako/runtime.py", line 835, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/mako/runtime.py", line 860, in _exec_template
    callable_(context, *args, **kwargs)
  File "/private/tmp/mako_modules/base.mako.py", line 79, in render_body
    __M_writer(unicode(self.content_wrap()))
  File "/private/tmp/mako_modules/base.mako.py", line 211, in render_content_wrap
    __M_writer(unicode(self.content()))
  File "/private/tmp/mako_modules/project/project_base.mako.py", line 57, in render_content
    __M_writer(unicode(next.body()))
  File "memory:0x10dc6ec50", line 123, in render_body

  File "memory:0x10dc6ec50", line 40, in render_node_settings

  File "memory:0x10dc6ec50", line 240, in render_render_node_settings

  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/mako/template.py", line 443, in render
    return runtime._render(self, self.callable_, args, data)
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/mako/runtime.py", line 803, in _render
    **_kwargs_for_callable(callable_, data))
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/mako/runtime.py", line 835, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/mako/runtime.py", line 860, in _exec_template
    callable_(context, *args, **kwargs)
  File "s3_node_settings_mako", line 35, in render_body
TypeError: 'Undefined' object is not callable
```